project/plugins.sbt

```diff
@@ -1,8 +1,14 @@
-resolvers += Resolver.url("hmrc-sbt-plugin-releases",
-  url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
+resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(
+  Resolver.ivyStylePatterns)
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.7.0")
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "0.9.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.13.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.15.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.14.0")
+
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
```

Comment:
We are bringing entire play as a dependency + the plugin but we only need `play-json`